mysql-test/main/mysqltest-break.test

```diff
@@ -0,0 +1,87 @@
+--echo #
+--echo # Test "break" command
+--echo #
+
+# Break in a single loop
+
+let $cnt= 4;
+while($cnt > 1)
+{
+  dec $cnt;
+  break;
+  --echo Break did not stop a single loop
+}
+
+# Break stops inner loop
+
+let $outer= 4;
+while($outer > 1)
+{
+  let $inner= 4;
+  while($inner > 1)
+  {
+    if($outer == 2)
+    {
+      --echo OK
+    }
+    if($inner == 2)
+    {
+      break;
+    }
+    dec $inner;
+  }
+  dec $outer;
+}
+
+
+# Break stops outer loop
+let $inner= 4;
+let $outer= 4;
+while($outer > 1)
+{
+  break;
+  while($inner > 1)
+  {
+    dec $inner;
+    --echo Outer loop`s break did not stop inner loop
+  }
+  dec $outer;
+}
+
+# Break stops loop in if
+let $cnt= 4;
+if($cnt > 1)
+{
+  while($cnt)
+  {
+    break;
+  }
+  dec $cnt;
+}
+
+if($cnt != 3)
```

Comment:
Well, this is good for me too. I'm good with and `echo $cnt`, and if($cnt != 3), and even both.
But at least some check should be there